src/main/java/io/github/mooy1/infinitylib/misc/BlockUtils.java

```diff
@@ -95,10 +103,10 @@
      *
      * @param start the block to start
      * @param treeType the type of the tree to get
-     *
      * @return a set of the blocks contained in the tree
      */
-    public static Set<Block> getTree(Block start, TreeType treeType) {
+    @Nonnull
+    public static Set<Block> getTree(@Nonnull Block start, @Nonnull TreeType treeType) {
```

Comment:
now this cant be get or default or itll default to all mats :notlikethis: id just do a if null return empty set